### PR TITLE
Opening our Slack for the community

### PR DESCRIPTION
**Pull request by rberenguel**
Slack

### CODE REVIEW

File: templates/header.html [hunk 1]
```diff
@@ -47,17 +47,18 @@
...
-                                                ['/job-offers', 'Job Offers'],
-					        ['/association/info', 'Association'],
+                                            ['/monthly-meetups', 'Monthly Meetups'],
+                                            ['/coding-dojos', 'Coding Dojos'],
+                                            ['/pyladies-bcn', 'PyLadies BCN'],
+                                            ['/pydatabcn-2017', 'PyDataBCN 2017'],
+                                            ['/pyday-bcn-2019', 'PyDay BCN 2019'],
+                                            ['/coc', 'Code of Conduct'],
+                                            ['/job-offers', 'Job Offers'],
+                              					    ['/association/info', 'Association'],
```
Comment (DZPM) [inline templates/header.html]: Too many tabs :-p

Comment (rberenguel) [reply templates/header.html]: Yeah... I'm actually doing a local POC of moving to Hugo and this looks horrible with their menu set up as well. I suggest removing (at some point) old events and keep only the next one, for a start

Comment (ifosch) [reply templates/header.html]: Or maybe having a main `Events` section...

Comment (ifosch) [reply templates/header.html]: Indentation also makes the code a little bit weird too.

**Pull request merged**